### ISSUE TITLE
PSR2.ControlStructures.SwitchDeclaration change between 3.1.1 and 3.2.2

### ISSUE THREAD

**Issue by PatrickRose** [squizlabs/PHP_CodeSniffer#1854]
```php
<?php


switch ($something)
{
        case 'Foo':
                do_something();
                break;

        case 'Bar':
                do_something();
                break;  
}
```

When I run this with PHPCS 3.1.1, I get no errors but when I run this with PHPCS 3.2.2 I get

```
$ phpcs tmp.php --sniffs=PSR2.ControlStructures.SwitchDeclaration

FILE tmp.php
--------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
--------------------------------------------------------------------------------------------
  8 | ERROR | [x] Terminating statement must be indented to the same level as the CASE body
 12 | ERROR | [x] Terminating statement must be indented to the same level as the CASE body
--------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------

Time: 102ms; Memory: 8Mb
```

This happens with tabs or space indentation (with tab width is set to 4).

**Comment by gsherwood** [comment]
I get exactly the same errors on PHPCS 3.1.1 and 3.2.2. I also tested master and 2.9.1 and get the same errors there. All versions report the terminating statement error if those case statements are indented 2 levels instead of 1 level deep.

If I change the indents to 1 tab indent each, I get no errors about terminating statements, but do get errors about tab indents.

If I change the indents to 4 spaces each, I get no errors about terminating statements or indents.

Do you have any more info that would help me replicate what you are seeing?

**Comment by PatrickRose** [comment]
Looks like I wasn't right in my previous comment:

| Indentation | Version | Result                                                                    |
|-------------|---------|---------------------------------------------------------------------------|
| Space       | 3.1.1   | Terminating statement must be indented to the same level as the CASE body |
| Space       | 3.2.2   | Terminating statement must be indented to the same level as the CASE body |
| Tabs        | 3.1.1   | No errors                                                                 |
| Tabs        | 3.2.2   | Terminating statement must be indented to the same level as the CASE body |

Based on what I can see in the [PSR2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#52-switch-case) standard, the 3.1.1 tab behaviour was correct?

**Comment by PatrickRose** [comment]
Just checked 2.9.1 and I'm getting the same behaviour as 3.1.1 (spaces is an error, tabs are not)

**Comment by PatrickRose** [comment]
Ah, found it. Our standard uses the PSR2 standard, but for some reason when I set this up 2 years ago I set the indent to 1. I'm guessing I was working around something.
